timetracker/vms/forms.py

```diff
@@ -130,6 +131,27 @@ def save(self, user):
             agency=agency
         )
 
+class CreateStaffEmployeeForm(forms.Form):
+    """
+    Form to create staffing employee.
+    """
+    agency_field=forms.ModelChoiceField(required=True,
+    queryset=models.StaffingAgency.objects.all(),
+    empty_label=None)
+
+    def save(self, user):
+        agency = self.cleaned_data['agency_field']
+        agency_object=models.StaffingAgency.objects.get(name=agency)
```

Comment:
There's no need for this. The `ModelChoiceField` returns an instance of the model already, so `self.cleaned_data['agency_field']` *is* a `StaffingAgency` instance.